packages/extension-chakra-storefront/src/components/footer/index.jsx

```diff
@@ -61,11 +50,15 @@ const Footer = ({...otherProps}) => {
     }
 
     return (
-        <Box as="footer" {...styles.container} {...otherProps}>
-            <Box {...styles.content} as="section">
-                <StylesProvider value={styles}>
-                    <HideOnMobile>
-                        <SimpleGrid columns={4} spacing={3}>
+        <Box as="footer" css={styles.container} {...otherProps}>
+            <Box css={styles.content} as="section">
+                <SimpleGrid columns={{base: 1, lg: 4}} gap={{base: 0, lg: 3}}>
+                    <GridItem colSpan={{base: 1, md: 3}}>
+                        <SimpleGrid
+                            columns={{base: 1, lg: 3}}
+                            gap={{base: 0, lg: 3}}
+                            display={{base: 'none', lg: 'grid'}}
```

Comment:
Is this where you are replacing the hideondesktop and hideonmobile implementation. Is there any issues that might arise with accessibility or server-side rendering and hydration with this? or is it all positives using a responsive design?

Reply:
Nothing comes up as far as I know.